software/distro/setup/planktoscope-app-env/PlanktoScope/install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
```

Comment:
If your'e stuck on https://github.com/PlanktoScope/PlanktoScope/actions/runs/14386498419/job/40343129723#step:15:3108 ("/run/os-setup/software/distro/setup/planktoscope-app-env/setup.sh: line 50: /run/os-setup/software/distro/setup/planktoscope-app-env/PlanktoScope/install.sh: Permission denied"), probably the reason is that `.../PlanktoScope/install.sh`'s file permissions are `.rw-r--r--` instead of `.rwxr-xr-x`.